miniapp/USERGUIDE.md

```diff
@@ -324,6 +330,28 @@ val userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher {
 
 // set UserInfoBridgeDispatcher object to miniAppMessageBridge
 miniAppMessageBridge.setUserInfoBridgeDispatcher(userInfoBridgeDispatcher)
+
+val chatBridgeDispatcher = object : ChatBridgeDispatcher {
+
+    override fun sendMessageToContact(
+        message: MessageToContact,
+        onSuccess: (contactId: String?) -> Unit,
+        onError: (message: String) -> Unit
+    ) {
+        // Check if there is any contact in HostApp
+        // .. .. ..
+        if (hasContact) {
+            // You can show a contact selection UI for picking a single contact.
+            // .. .. ..
+            onSuccess(contactId) // allow miniapp to invoke the contact id where message has been sent.
```

Comment:
from spec it returns with `messageId`, not `contactId`.

Reply:
Oh really? sorry I didn't know if the spec has been changed and `messageId` has been introduced instead of `contactId`.
Can you please share any link / reference of the spec? However, I am reading from this spec: https://confluence.rakuten-it.com/confluence/x/Xh2Mk attached in the Jira where it still says `contactId`, and so, demo app is invoking the contact id. I haven't seen anything about message id 🤔

Reply:
oops I'm sorry my mistake. I read the spec wrongly 🤯

Reply:
Ah, that's okay. Things are going busy with lots of spec :D